Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.45491&attempt_id=c9dc0553-79cb-4b02-ba71-67faf6525dae)

Attempted: wt3

<details><summary>Partial log (click to expand)</summary><p>

```
make[3]: *** [../gen/make.rules:112: ../temp/boot/jrd/gds.o] Error 1
make[3]: Leaving directory '/build/Firebird-2.5.7.27050-0/gen'
make[2]: *** [Makefile:233: gpre_boot] Error 2
make[2]: Leaving directory '/build/Firebird-2.5.7.27050-0/gen'
make[1]: *** [Makefile:227: ../gen/firebird/bin/gpre_static] Error 2
make[1]: Leaving directory '/build/Firebird-2.5.7.27050-0/gen'
make: *** [Makefile:6: firebird] Error 2
builder for '/nix/store/bzivcvh2hnafmqga4x0ifz1hwvv8wqw1-firebird-2.5.7.27050-0.drv' failed with exit code 2
cannot build derivation '/nix/store/s81kqwhw5d903hsgmf2i0l2mxay3yj4c-wt-3.3.11.drv': 1 dependencies couldn't be built
error: build of '/nix/store/s81kqwhw5d903hsgmf2i0l2mxay3yj4c-wt-3.3.11.drv' failed
```
</p></details>

--- a/pkgs/development/libraries/wt/default.nix
+++ b/pkgs/development/libraries/wt/default.nix
@@ -43,8 +43,8 @@ let
     };
 in {
   wt3 = generic {
-    version = "3.3.10";
-    sha256 = "1y25mhghgbgjgycpny0x4z95xn98q0wraab1c5gkwnay097bgwdy";
+    version = "3.3.11";
+    sha256 = "1s1bwg3s7brnspr9ya1vg5mr29dbvhf05s606fiv409b7ladqvxq";
   };
 
   wt4 = generic {

--- a/pkgs/development/libraries/wt/default.nix
+++ b/pkgs/development/libraries/wt/default.nix
@@ -48,7 +48,7 @@ in {
   };
 
   wt4 = generic {
-    version = "4.0.3";
-    sha256 = "01xch5dkpcanwhr515236wa9mdmnq2a2j13dn7smyhwzqgbpknsg";
+    version = "4.0.4";
+    sha256 = "17kq9fxc0xqx7q7kyryiph3mg0d3hnd3jw0rl55zvzfsdd71220w";
   };
 }

